lib/amnesia/storage.rb

```diff
@@ -16,14 +16,26 @@ def set(key, value)
       File.write(filename, entry, mode: 'a+')
     end
 
+    def delete(key)
+      set(key, '')
+    end
+
     def get(key, index_entry: nil)
       return record_from_index(index_entry) unless index_entry.nil?
 
       record_from_scan(key)
     end
 
     def parse_record(raw_record)
-      raw_record.split(',', 2)[1]
+      value = raw_record_value(raw_record)
+
+      return '(nil)' if value.nil? || value.empty?
```

Comment:
We should move it to the CLI. The storage should not care about how it will be displayed on cli